ignite/distributed/utils.py

```diff
@@ -737,3 +740,11 @@ def download_dataset():
 
     if current_rank == rank:
         barrier()
+
+
+def _rank_not_in_group(group: Optional[dist.ProcessGroup | "hvd.ProcessSet"]) -> bool:
+    """Check if the current process's rank is not in a given group."""
+    if has_hvd_support and isinstance(group, hvd.common.process_sets.ProcessSet):
```

Comment:
As group can be None we should handle that as well

Reply:
`dist._rank_not_in_group(group)` handles that. Doesn't it?

Reply:
In case when we have HVD and group is None, we should not enter `dist._rank_not_in_group(group)` ?